### DISCUSSION TITLE
Translation Progress Indicator

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4795] [Feature Requests]
We need a stopgap solution here, as it's too difficult to know what has been started/completed. I am fine with fetching all the relational data and then having the interface calculate based on "fields with a value". Later, we can add a more detailed indicator that shows the percentage of fields that _pass validation_ (a more accurate metric).

Progress can be displayed as numbers (eg: `0/4` or `3/4`)... or as a stylized progress bar.

In the future we can optimize this to use the "computed values" metadata... so we don't have to fetch **all** the relational data.

**Comment by Nitwel** [comment]
Is this progess indicator meant for the translation interface or am I getting this wrong here?
![grafik](https://user-images.githubusercontent.com/22577866/108378344-3389bd80-7205-11eb-8d01-1ecf3b9ff8d5.png)

**Comment by benhaynes** [comment]
Correct. Official mockup below. The difficult part will be calculating what is "done"... but as long as we have something it will be helpful.

![Screen Shot 2021-02-18 at 10 23 34 AM](https://user-images.githubusercontent.com/522079/108378865-62873b80-71d3-11eb-9592-dd75ac0f3d1e.png)

**Comment by Nitwel** [comment]
My first approach would be to just show if an related item already exists / if it has been translated at all already.
Having such detailed information would be interesting but could get complex.

**Comment by benhaynes** [comment]
Yeah, that is a good first step! Once we have proper validation we can just check the percentage of fields that pass the check.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
